src/frameworks/opengles/gles1_on_gl2.rs

```diff
@@ -669,6 +669,9 @@ impl GLES for GLES1OnGL2 {
         assert!(target == gl21::TEXTURE_2D);
         gl21::TexParameterf(target, pname, param);
     }
+    unsafe fn TexEnvi(&mut self, target: GLenum, pname: GLenum, param: GLint) {
+        gl21::TexEnvi(target, pname, param);
```

Comment:
Can you add an assert for the `pname` values? Because OpenGL ES 1.1 and OpenGL 2.1 aren't exact supersets or subsets of eachother, I try to check the app doesn't use something unsupported.

What I usually do is go to https://registry.khronos.org/OpenGL-Refpages/es1.1/xhtml/ and https://registry.khronos.org/OpenGL-Refpages/gl2.1/ so I can compare the two versions of the function, and compare them to see what the common subset of enum values is. Then I assert that it's in that common subset.

Reply:
The only difference appears to be that ES 1.1 has GL_COORD_REPLACE_OES, while 2.1 has GL_COORD_REPLACE. I can't seem to find any info about the difference between the two, though.

Reply:
Interestingly, these parameter names both have the same value (0x8862 or 34914). Could they be considered equivalent?

Reply:
I think so? I am not sure why some things have that suffix in OpenGL ES 1.1 but not in OpenGL 2.1. It might be because OpenGL ES 1.1 is based on OpenGL 1.5, which maybe didn't have some of these enum values.

I should probably add a note about it somewhere, but I assume in many places that enum values match between ES and normal GL.

Reply:
Would the assert still be needed?

Reply:
I would still prefer to have the assert. I want to ensure that if the app uses something we don't expect, we catch it right away, rather than it causing e.g. an OpenGL error that will only be caught the next time the app calls `glGetError()`.

Reply:
Actually, I need to implement part of the `glTexEnv` family now for the app I'm currently trying to get running, so I may as well implement the full thing and save you the effort.